### PR TITLE
[NUI] TextField rename TextField_SelectNone to be SelectNone

### PR DESCRIPTION
**Pull request by Ali-Alzyoud**
Class: TextField
Rename TextField_SelectNone to SelectNone
Remove unneeded type cast

### Description of Change ###
Rename SelectNone function.

### API Changes ###

 - ACR:

### CODE REVIEW

File: src/Tizen.NUI/src/public/BaseComponents/TextField.cs [hunk 1]
```diff
@@ -1101,6 +1101,22 @@ public bool EnableGrabHandlePopup
...
+        /// <since_tizen> 8 </since_tizen>
+        /// This will be public opened in tizen_6.0 after ACR done, Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string SelectedText
+        {
+            get
+            {
+                string temp;
+                GetProperty(TextField.Property.SELECTED_TEXT).Get(out temp);
+                return (string)temp;
```
Comment (idkiller) [inline src/Tizen.NUI/src/public/BaseComponents/TextField.cs:1116]: You don't need to cast the string to string again.

Comment (Ali-Alzyoud) [reply src/Tizen.NUI/src/public/BaseComponents/TextField.cs]: thank you, cast is removed

**Comment by JoogabYun** [comment]
please resolve conflicts

**Comment by Ali-Alzyoud** [comment]
With this revert #aa10d89 this patch can not be applied.

**Comment by JoogabYun** [comment]
Oh.. it reverted again.

**Comment by JoogabYun** [comment]
please check this patch. can you merge?

**Comment by Ali-Alzyoud** [comment]
> please check this patch. can you merge?

Done

**Pull request merged**